.github/workflows/push_tests.yml

```diff
@@ -103,3 +103,7 @@ jobs:
         run: |
           cd client && make smoke-test
           ./node_modules/codecov/bin/codecov --yml=../.codecov.yml --root=../ --gcov-root=../ -C -F frontend,javascript,smokeTest
+      - name: Smoke tests (with annotations feature)
```

Comment:
I couldn't find where `e2eAnnotations.test.ts` was being run, so I added it to `push_tests.yml`.

Reply:
It's be a while since I originally filed the story behind this PR, so can you remind me what UI features this is actually testing? Is it "gene sets"?  Since user annotations have been removed, that's all I can imagine it is doing. 

I'm not privy to the history behind why these two smoke test groups were split in the first place. If there's no good reason (any more), I'm wondering if it would be worth merging them together to simplify things?

Reply:
> t's be a while since I originally filed the story behind this PR, so can you remind me what UI features this is actually testing? Is it "gene sets"? Since user annotations have been removed, that's all I can imagine it is doing.

At this point, the annotations tests are covering:
- the possible actions related to genesets (creation, coloring, editing, undo/redo, etc)
- differential expression (not sure why that's being done here but I suppose it relates to geneset creation)
- stacked bar graph renders (relates to the categorial labels so I guess it belongs here)
- check the number of cells associated with a label

> I'm not privy to the history behind why these two smoke test groups were split in the first place. If there's no good reason (any more), I'm wondering if it would be worth merging them together to simplify things?

Me neither - I think merging the tests makes sense.

Reply:
Great, thanks for listing those. Those all seem like core features to test so merging all the smoke tests together still sounds right. @czi-sci-single-cell-eng Anyone disagree or have insights as to why they were split in the first place?

Reply:
I think they were originally split prior to the repo split. Because annotations were only available if specified in the config (usually for the desktop) we wanted to make sure the general flow worked with and without them. It should be safe to merge at this point

Reply:
ditto, safe to merge